go.sum

```diff
@@ -864,6 +864,10 @@ github.com/rancher/remotedialer v0.2.6-0.20200401224317-04cc22157b83/go.mod h1:d
 github.com/rancher/rke v1.1.5-rc3/go.mod h1:Vs49ny2u80SAQwbsCf0RUjixT2aTOj6rZWwQKcpE37c=
 github.com/rancher/rke v1.1.7-0.20200909222504-7a68adea9093 h1:J8+YwwnflmS/dHosZWZeaivueW2p/f/WMaTHBrsGohw=
 github.com/rancher/rke v1.1.7-0.20200909222504-7a68adea9093/go.mod h1:ixdG4yOKf6yJxmj/wqHJniMHOgParBzS1cQ9u3Mq4no=
+github.com/rancher/rke v1.1.8-rc1 h1:HrllsFLWneSfwW9ZJP8mpgIEkB8maTrc3ufOw4lDY7A=
```

Comment:
Is this v1.1.8-rc1 stuff in here okay, or should it be v1.1.9-rc1 only?

Reply:
updated